metabase/metabase
# Allow image URL preview in object detail

This was planned work as part of [an old object detail epic](https://www.notion.so/metabase/Make-it-easier-to-get-to-and-navigate-through-individual-record-view-e475d6a016794b1cbdee0334058802e5?pvs=4#7d4764dae8ba4f02ab9bb4a9522211be)


[Designs](https://www.figma.com/file/or0R2ynrqOhN2RBv6uH5Qv?embed_host=notion&kind=&node-id=835%3A37990&viewer=1)

Relates to https://github.com/metabase/metabase/issues/22119

### Description

It's kind of weird that our table visualization supports image URL previews, but our detail component does not. This displays images in the object detail for any `imageURL` or `avatarURL` type columns.

![detail images](https://github.com/metabase/metabase/assets/30528226/dce6131b-5a93-4ad9-837e-9b7d9ee6bd62)


Modal | Question Visualization | Dashboard Visualization
---|---|---
![Screen Shot 2023-06-12 at 4 19 21 PM](https://github.com/metabase/metabase/assets/30528226/75cd65f2-21de-4904-bd03-ba1a8ce46f99) | ![Screen Shot 2023-06-12 at 4 19 06 PM](https://github.com/metabase/metabase/assets/30528226/609c6e73-7a34-4bbb-9795-d81ecda91436) | ![Screen Shot 2023-06-12 at 4 18 50 PM](https://github.com/metabase/metabase/assets/30528226/2b497650-4900-48b7-bb06-0a21eef94b5d)


### How to verify

- Load a dataset with image URLs [like this one](https://www.notion.so/metabase/Pokemon-TCG-66dd0b22e43c486ba296c2b06b360c7e). 
- Ensure that one or more columns are marked as image urls or avatar urls on your table or model
- See that the images display in the detail modal.

### Checklist
